ign-rendering6.yaml

```diff
@@ -0,0 +1,25 @@
+repositories:
+  ign-cmake:
+    type: git
+    url: https://github.com/ignitionrobotics/ign-cmake
+    version: ign-cmake2
+  ign-common:
+    type: git
+    url: https://github.com/ignitionrobotics/ign-common
+    version: ign-common4
+  ign-math:
+    type: git
+    url: https://github.com/ignitionrobotics/ign-math
+    version: ign-math6
+  ign-plugin:
+    type: git
+    url: https://github.com/ignitionrobotics/ign-plugin
+    version: ign-plugin1
+  ign-rendering:
+    type: git
+    url: https://github.com/ignitionrobotics/ign-rendering
+    version: main
+  ign-utils:
+    type: git
+    url: https://github.com/ignitionrobotics/ign-utils
+    version: ign-utils1
```

Comment:
I'm also not seeing this in the `packages.apt` for `ign-rendering`.

Reply:
Marking as resolved based on [this discussion](https://github.com/ignition-tooling/gazebodistro/pull/32#discussion_r605969706)